src/commands/publish.ts

```diff
@@ -396,6 +402,49 @@ async function handleReleaseBranch(
   }
 }
 
+/**
+ * Run an external post-release command
+ *
+ * The command is usually for bumping the development version on master or
+ * cleanup tasks.
+ *
+ * @param newVersion Version being released
+ * @param postReleaseCommand Custom post-release command
+ */
+export async function runPostReleaseCommand(
+  newVersion: string,
+  postReleaseCommand?: string
+): Promise<boolean> {
+  let sysCommand: string;
+  let args: string[];
+  if (postReleaseCommand !== undefined && postReleaseCommand.length === 0) {
+    // Not running post-release command
+    logger.warn('Not running the pre-release command: no command specified');
+    return false;
+  } else if (postReleaseCommand) {
+    [sysCommand, ...args] = shellQuote.parse(postReleaseCommand);
+  } else if (existsSync(DEFAULT_POST_RELEASE_SCRIPT_PATH)) {
+    sysCommand = '/bin/bash';
+    args = [DEFAULT_POST_RELEASE_SCRIPT_PATH];
+  } else {
+    // Not running post-release command
+    logger.warn(
```

Comment:
Would suggest to make these this message and the one above `info` (or lower) instead of `warn`, most of the people will not want to see a warning about a thing they don't really care about